src/textual/_layout.py

```diff
@@ -28,16 +36,55 @@ def __repr__(self) -> str:
         return f"<{self.name}>"
 
     @abstractmethod
-    def arrange(
-        self, parent: Widget, size: Size, scroll: Offset
-    ) -> tuple[list[WidgetPlacement], set[Widget]]:
+    def arrange(self, parent: Widget, size: Size) -> ArrangeResult:
```

Comment:
Dropped `scroll` parameter which wasn't used. Added a type alias for the return type.